monkey/monkey_island/cc/environment/environment_singleton.py

```diff
@@ -39,12 +40,17 @@
         user_store.UserStore.set_users(env.get_auth_users())
 
 
-try:
-    config = EnvironmentConfig.get_from_file()
-    __env_type = config.server_config
-    set_env(__env_type, config)
-    # noinspection PyUnresolvedReferences
-    logger.info('Monkey\'s env is: {0}'.format(env.__class__.__name__))
-except Exception:
-    logger.error('Failed initializing environment', exc_info=True)
-    raise
+def initialize_from_file(file_path):
+    try:
+        config = EnvironmentConfig(file_path)
+
+        __env_type = config.server_config
+        set_env(__env_type, config)
+        # noinspection PyUnresolvedReferences
+        logger.info('Monkey\'s env is: {0}'.format(env.__class__.__name__))
+    except Exception:
+        logger.error('Failed initializing environment', exc_info=True)
+        raise
+
+
+initialize_from_file(DEFAULT_SERVER_CONFIG_PATH)
```

Comment:
Why is this called here?

Reply:
A comment explaining this has been added in a later commit. 93d0fcc7ddf14602d99cd789148ff9cedab5ab27